doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -18,17 +18,17 @@ Overview:
 
 #. `Design`_
```

Comment:
There are still several mentions of contributor license agreements in this document. Should those sections be removed in this PR?

Reply:
Sheesh, there sure are!  Who reviewed this last time anyway...  (◔_◔)  Apparently, I was grepping on too narrow of a pattern.